parts/linux/cloud-init/artifacts/components.json

```diff
@@ -8,512 +10,18 @@
         "1.8.22"
       ]
     },
+
+    //new
     {
-      "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-server:*",
+      "name": "nvidia-cuda",
+      "type": "containerimages //extend to OCI |Tar|tgz when merging with packages,
+      "action": "cacheandfetch|fetchonly|config",  //where cacheandfetch is cached at build time and also to fetch at provisioning time if not cached. fetchonly is only fetching at provisioning time but not caching. config is neither cache nor fetch but just for storing data for other purpose. Open to change the names of the values.
```

Comment:
@ganeshkumarashok do you still a config for passing the value to the CSE? If not I can remove this. Though even it's needed, we can still discuss if there are better ways to put in the component.json